crates/volta-core/src/shim.rs

```diff
@@ -73,7 +73,7 @@ pub fn create(shim_name: &str) -> Fallible<ShimResult> {
     #[cfg(windows)]
     windows::create_git_bash_script(shim_name)?;
 
-    match symlink_file(executable, shim) {
+    match copy_shim(executable, shim) {
```

Comment:
As mentioned above, I suspect what we want here are two separate implementations: One for `#[cfg(windows)]` and one for `#[cfg(not(windows))]`, rather than changing the helper that's used. We've already got some custom Windows logic (creating the git bash script), so it's probably the most straightforward at this point to completely separate them.